rulesets/resources/ES_ZERHruleset.rb

```diff
@@ -959,13 +959,13 @@ def self.set_appliances_clothes_washer_reference(orig_bldg, new_bldg)
     label_usage = reference_values[:label_usage]
     capacity = reference_values[:capacity]
 
-    # Override values?
+    # Override efficiency values equal to "Std 2018=Present" Standard if clothes washer present in the Rated Home
     if not orig_bldg.clothes_washers.empty?
       clothes_washer = orig_bldg.clothes_washers[0]
       id = clothes_washer.id
       location = clothes_washer.location.gsub('unvented', 'vented')
 
-      if [ESConstants::SFNationalVer3_2, ESConstants::MFNationalVer1_2, ZERHConstants::SFVer2, ZERHConstants::MFVer2].include? @program_version
+      if [ESConstants::SFNationalVer3_2, ESConstants::SFNationalVer3_3, ESConstants::MFNationalVer1_2, ESConstants::MFNationalVer1_3, ZERHConstants::SFVer2, ZERHConstants::MFVer2].include? @program_version
```

Comment:
It looks like this is basically the only reference left to `@program_version` in this ruleset. Can you figure out a way to remove it and incorporate the logic in the program lookup tsv's instead?